analytics/js_tracking_api.rst

```diff
@@ -46,9 +46,9 @@ This code should be added just before you want to use the API. Additionally snip
       _paq.push(["trackPageView"]);
       _paq.push(["enableLinkTracking"]);
       (function() {
-        var u="//{$PIWIK_URL}/";
+        var u="ppms.example.com";
```

Comment:
I think slashes have to stay to build URLs. So it should be `"//ppms.example.com/"`.